docs/prover/cli/options.md

```diff
@@ -793,29 +793,117 @@ Advanced options that control control flow splitting
 
 See [here](control-flow-splitting) for an explanation of control flow splitting.
```

Comment:
I would shorten "Advanced options that control control flow splitting" to "Control flow splitting options".  It also looks like the header levels are inconsistent; take a look at the generated table of contents at the top of the page.

Reply:
header: right, much better ...

hm, the toc looks okay-ish to me -- "Control flow splitting options" is on one level with the other groups ("Options that control the soldiity compiler", "Advanced Options", etc .. though whether Advanced options is a great category might be subject for discussion ..)